### ISSUE TITLE
setup command for windows

### ISSUE THREAD

**Issue by jarvet** [stripe/python-interview-prep#5]
```bash
virtualenv interview_env --python=python2.7
source interview_env/bin/activate
```
seems the command above won't work for Windows.

For Windows users (I'm Win 10), I'd suggest to try: 
```bash
virtualenv interview_env --python=2.7
call interview_env/Scripts/activate.bat
```

**Comment by Beatricce** [comment]
Yes, it seems like a lot of us use PowerShell, so the command in the README file doesn't work. For reference: https://dev.to/shriekdj/how-to-create-and-activate-the-virtual-environment-for-python3-project-3g4l. Also, for me `.\interview_env\Scripts\activate.bat.` worked
